src/modules/flow/iio/iio.json

```diff
@@ -421,8 +421,8 @@
            "name": "use_device_default_scale"
          },
          {
-             "data_type": "float",
-             "default": 0,
+             "data_type": "direction-vector",
+             "default": {},
```

Comment:
@bdilly AFAIU he wants a scale, like "When I read RED, multiply it by 0.3".

In that sense I wonder if we do really need this, given that the output RGB packet can hold the maximum value of components, then we can do the scale/map afterwards.